x-pack/test/functional/services/cases/common.ts

```diff
@@ -63,7 +63,6 @@ export function CasesCommonServiceProvider({ getService, getPageObject }: FtrPro
       const radioGroup = await testSubjects.find(testSubject);
       const label = await radioGroup.findByCssSelector(`label[for="${value}"]`);
       await label.click();
-      await this.assertRadioGroupValue(testSubject, value);
```

Comment:
Note to not forget to make sure that this function is not used by other tests that expect the function to assert.

Reply:
just added wait time before checking assert as this function is used in multiple places